src/boilerplate/dispatcher.c

```diff
@@ -127,7 +127,7 @@ void apdu_dispatcher(command_descriptor_t const cmd_descriptors[],
 
     G_dispatcher_context.read_buffer = buffer_create(cmd->data, cmd->lc);
 
-    if (cmd->p1 != 0 || cmd->p2 > 1) {
+    if (cmd->p2 > 1) {
```

Comment:
nit : If this 1 value corresponds to the protocol version maybe a comment or a define could be nice ? :)

Reply:
Good point, added a constant in 4b199cef7deba7789298ed99b78b0ea4898d71de (and rebase to the current `develop`).